python/doc/examples/reliability_sensitivity/reliability/plot_multi_form.py

```diff
@@ -0,0 +1,139 @@
+graphModel.setXTitle(r'$x_2$')
+graphModel.setTitle(r'Isolines of the model : $Y = f(X)$')
+view = otv.View(graphModel)
+
+
+# %%
+# We create random vectors for the input and output variables :
+X = ot.RandomVector(dist)
+Y = ot.CompositeRandomVector(f, X)
+
+
+# %%
+# The failure domain :math:`\mathcal{D}` is :math:`\mathcal{D} = \{ x=(x_1, x_2) \in \mathbb{R}^2 / f(x) \geq 0 \}`
+failureEvent = ot.ThresholdEvent(Y, ot.Less(), 0.0)
+
+
+# %%
+# We shall represent the failure domain event using a :class:`~openturns.Contour` object.
+nx, ny = 25, 25
+xx = ot.Box([nx], ot.Interval([-8.0], [8.0])).generate()
+yy = ot.Box([ny], ot.Interval([-8.0], [8.0])).generate()
+inputData = ot.Box([nx,ny], ot.Interval([-8.0, -8.0], [8.0, 8.0])).generate()
+outputData = f(inputData)
+mycontour = ot.Contour(xx, yy, outputData, ot.Point([0.0]), ot.Description(["0.0"]))
+mycontour.setColor("black")
+mycontour.setLineStyle("dashed")
+graphModel.add(mycontour)
+view = otv.View(graphModel)
+
+# %%
+# In the physical space the failure domain boundary is the dashed black curve. We recall that one of
+# the steps of the FORM method is to find the closest point of the failure domain boundary to the origin.
+# Here we see that the symmetry of the domain implies that two points exist, one in the :math:`x_1 \geq 0` half-space and the other in the :math:`x_1 \leq 0` half-space.
+
+
+# %%
+# We build the :class:`~openturns.MultiFORM` algorithm in a similar fashion as the :class:`~openturns.FORM` algorithm. We choose an optimization solver, here the Cobyla solver, and a starting point, the mean
+# of the distribution `dist`.
+solver = ot.Cobyla()
+starting_pt = dist.getMean()
+algo = ot.MultiFORM(solver, failureEvent, starting_pt)
+
+
+# %%
+# We are ready to run the algorithm and store the result in the :class:`~openturns.MultiFORM` class :
+algo.run()
+result = algo.getResult()
+
+# %%
+# We have access to the results with the getFORMResultCollection method which produces a collection of :class:`~openturns.FORMResult`s: 
+coll = result.getFORMResultCollection()
+
+# %%
+# The length of this collection is the number of design points : 
+n_design_pts = len(coll)
+print("Number of design points :", n_design_pts)
+
+
+# %%
+# We have access to the design points with the getPhysicalSpaceDesignPoint method for each element of
+# the collection `coll`.
+designPointStandardSpace1 = coll[0].getPhysicalSpaceDesignPoint()
+designPointStandardSpace2 = coll[1].getPhysicalSpaceDesignPoint()
+print(coll[0].getPhysicalSpaceDesignPoint())
+print(coll[1].getPhysicalSpaceDesignPoint())
+
+# %%
+# We visualize them on the previous graph with red circle dots.
+cloud = ot.Cloud(ot.Point([designPointStandardSpace1[0]]), ot.Point([designPointStandardSpace1[1]]))
+cloud.setColor("red")
+cloud.setPointStyle("fcircle")
+cloud.setLegend("design point no. 1")
+graphModel.add(cloud)
+
+cloud = ot.Cloud(ot.Point([designPointStandardSpace2[0]]), ot.Point([designPointStandardSpace2[1]]))
```

Comment:
there should be no nedd for Point here, just "[val]" should be enough

Reply:
This and previous remarks have been done in the next commit.